docs/modules/user-guide/partials/proc_defining-endpoints.adoc

```diff
@@ -55,9 +59,9 @@ components:
 ----
 ====
 
-Here, there are two container, each defining a single endpoint. Endpoint is an accessible port that can be made accessible inside the workspace or publicly from the UI. Each endpoint has a name and port, which is the port on which certain server running inside the container is listening. The following are a few attributes that you can set on the endpoint:
+This example has two containers, each defining a single endpoint. You can make the endpoint an accessible port inside the workspace or a public port from the UI. Each endpoint has a name and port. The server running inside the container listens to this port. See the following attributes that you can set on the endpoint:
```

Comment:
* Meaning change:
   _Endpoint is an accessible port..._

   vs:
   _You can make the endpoint an accessible port..._

* Meaning change:
   _certain server running inside the container_

   vs:
   _The server running inside the container_

* Servers don't listen **to** ports; they listen **on** ports. The orig. was correct.

* The `{prod-short}` attribute here again doesn't mean `devfile`:
   * Eclipse Che can have TLS enabled -- not devfiles.
   * There is no such thing as "devfile user interface".

Reply:
@jc-berger, again marked as resolved but not really resolved.

* There's still the mention of UI -- what UI? Eclipse Che? Probably, in which case it needs to be made clear -- devfiles are app-agnostic.
* Servers don't run on ports. Servers listen on ports.

Reply:
Thought you wanted me to return to the original phrasing. I changed it to `The endpoint is an accessible port that can be made accessible inside the workspace or publicly from the user's UI.` to make it more agnostic. 

Let me know what you think about this revision: `Each endpoint has a name and port which is the port on which the server is listening.` Or we can go to my previous revision to communicate that the server runs in the container but listens on the port: `Each endpoint has a name and port. The server running inside the container listens to this port`

Reply:
> The endpoint is an accessible port that can be made accessible inside the workspace or publicly from the user's UI.

Unfortunately, that makes no sense because the "UI" in the orig. content referred to Eclipse Che UI. In the absence of Che in the doc, talking about a UI is confusing. See above -- the same thing applies to "UI" as to "cluster".

> Each endpoint has a name and port which is the port on which the server is listening.

That's OK, but the orig. text was actually clearer. So, let's just fix the lang. of the original but keep its wording:

_Each endpoint has a name and a port, which is the port on which the server running inside the container is listening._

Reply:
Made the change. Removed the reference to being accessible publicly from the UI.